### PR TITLE
[Fix_transport_mode] Support for transport param in  network config

### PR DESCRIPTION
**Pull request by mazinamdar**
Signed-off-by: mazinamdar <mazhar.inamdar@seagate.com>

```
cortx_setup network config --mode tcp --transport lnet
```

### CODE REVIEW

**Review by divya-vijay — APPROVED** [review]
Looks good

File: lr-cli/cortx_setup/commands/network/config.py [hunk 1]
```diff
@@ -135,7 +142,7 @@ def run(self, mode=None, network_type=None, interfaces=None,
             )
 
         if mode is not None:
-            mode_type = 'transport_type' if mode in ['lnet', 'libfabric'] else 'interface_type'
+            mode_type = 'interface_type'
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/network/config.py:145]: Change this to `conf_key`

File: lr-cli/cortx_setup/commands/network/config.py [hunk 2]
```diff
@@ -149,6 +156,21 @@ def run(self, mode=None, network_type=None, interfaces=None,
                 f'server_node>{machine_id}>network>data>{mode_type}',
                 mode
             )
+        if transport is not None:
+            mode_type = 'transport_type'
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/network/config.py:160]: Change this to `conf_key`

File: lr-cli/cortx_setup/commands/network/config.py [hunk 3]
```diff
@@ -149,6 +156,21 @@ def run(self, mode=None, network_type=None, interfaces=None,
...
+            self.logger.debug(
+                f"Set {mode_type} to {mode}"
+            )
+            PillarSet().run(
+                f'cluster/{node_id}/network/data/{mode_type}',
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/network/config.py:165]: Change this to `conf_key`

File: lr-cli/cortx_setup/commands/network/config.py [hunk 4]
```diff
@@ -149,6 +156,21 @@ def run(self, mode=None, network_type=None, interfaces=None,
...
+                f'{transport}',
+                local=True
+            )
+            Conf.set(
+                'node_config_index',
+                f'server_node>{machine_id}>network>data>{mode_type}',
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/network/config.py:171]: Change this to `conf_key`

**Review by nitin-seagate — APPROVED** [review]
LGTM

File: lr-cli/cortx_setup/commands/network/config.py (hunk 1 above)
Comment (mazinamdar) [reply lr-cli/cortx_setup/commands/network/config.py]: Done

File: lr-cli/cortx_setup/commands/network/config.py (hunk 2 above)
Comment (mazinamdar) [reply lr-cli/cortx_setup/commands/network/config.py]: Done

File: lr-cli/cortx_setup/commands/network/config.py (hunk 3 above)
Comment (mazinamdar) [reply lr-cli/cortx_setup/commands/network/config.py]: Done

File: lr-cli/cortx_setup/commands/network/config.py (hunk 4 above)
Comment (mazinamdar) [reply lr-cli/cortx_setup/commands/network/config.py]: Done

**Pull request merged**